x-pack/legacy/plugins/canvas/i18n/expression_types.ts

```diff
@@ -145,14 +145,15 @@ export const ArgTypesStrings = {
 };
 
 export const ExpressionDataSourceStrings = {
+  // Elasticsearch documents datasource/Elasticsearch basic datasource
   ESDocs: {
     getDisplayName: () =>
       i18n.translate('xpack.canvas.expressionTypes.datasources.esdocsTitle', {
-        defaultMessage: 'Elasticsearch raw documents',
+        defaultMessage: 'Elasticsearch documents',
```

Comment:
```suggestion
        defaultMessage: '{elasticsearch} documents',
        values: {
           elasticsearch: ELASTICSEARCH
        }
```

Reply:
This needs to be done for all of these messages that include `Elasticsearch`.  This is to identify that elasticsearch is a word that does not need translation, and it ensures we are consistent with spelling (Elasticsearch vs ElasticSearch vs ElAsTiCsEaRcH) and can globally change it in one location.

Reply:
I've added these in, thanks for letting me know